src/cron/cleanGiftedPlus.ts

```diff
@@ -0,0 +1,25 @@
+import { Cron } from './cron';
+import { User } from '../entity';
+
+const cron: Cron = {
+  name: 'clean-gifted-plus',
+  handler: async (con, logger) => {
+    logger.info('cleaning gifted plus membership...');
```

Comment:
no need to log this, we will see cron ran from the jobs panel, but if you want to leave it you can change it to `debug`

Reply:
Updated to `debug`